examples/yacat/yacat.py

```diff
@@ -77,10 +78,11 @@ async def compute_keyspace(context: WorkContext, tasks: AsyncIterable[Task]):
     """
     async for task in tasks:
         cmd = f"hashcat --keyspace " f"-a {HASHCAT_ATTACK_MODE} -m {args.hash_type} {args.mask}"
-        context.run("/bin/bash", "-c", cmd)
+        s = context.new_script(timeout=KEYSPACE_TIMEOUT)
+        s.run("/bin/bash", "-c", cmd)
 
         try:
-            future_result = yield context.commit(timeout=KEYSPACE_TIMEOUT)
+            future_result = yield s
```

Comment:
And again, let's use the command's result directly.

Reply:
and again... I'd prefer not to complicate this pull request further